CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+Version 291:
+
+* Test websocket with use_awaitable (#1920)
```

Comment:
No need to include the issue numbers (especially since they aren't linked).

Reply:
I do this so that the release notes will be easier to compile. If it's ok with you I'd prefer to leave them in at least until release time. What do you think?

Reply:
The release notes should be compiled from the commit log not the CHANGELOG